README.md

```diff
@@ -1,5 +1,40 @@
-# Node-RED Quantum
-Quantum computing functionality for Node-RED.
+# node-red-quantum
+
+[![platform](https://img.shields.io/badge/platform-Node--RED-red)](https://nodered.org)
+![NPM version]()
+![NPM]()
```

Comment:
```suggestion
[![platform](https://img.shields.io/badge/platform-Node--RED-red)](https://nodered.org)
[![CI Status](https://img.shields.io/github/workflow/status/louislefevre/node-red-contrib-quantum/Node.js%20CI)](https://github.com/louislefevre/node-red-contrib-quantum/actions/workflows/node.js.yml)
[![NPM](https://img.shields.io/npm/v/node-red-contrib-quantum)](https://www.npmjs.com/package/node-red-contrib-quantum)
[Node](https://img.shields.io/node/v/node-red-contrib-quantum)
```
- Added a build status badge.
- Filled in badge for NPM (it won't work right now, but it should do once its been deployed).
- Is the "NPM version" badge needed for anything different? The NPM badge shows the version anyway.
- Added Node badge.